build/Clean.ps1

```diff
@@ -0,0 +1,12 @@
+$root_dir = Resolve-Path "$PSScriptRoot\..\"
```

Comment:
Not sure if this was meant to be included. I typically do `git clean -fxd` locally. Build servers are from a clean image on every run.

Reply:
The files in the build folder are copied from Foundatio.AzureStorage, I do not know the purpose of this line exactly.

Reply:
Ah thanks for the heads up. I'm not sure how they got there. I'll remove those :)

Reply:
Cleaned up the azure repo :)